app/v2/controllers/ListHistoricUkPropertyPeriodSummariesController.scala

```diff
@@ -60,17 +59,17 @@ class ListHistoricUkPropertyPeriodSummariesController @Inject() (val authService
     authorisedAction(nino).async { implicit request =>
       implicit val ctx: RequestContext = RequestContext.from(idGenerator, endpointLogContext)
 
-      val rawData = ListHistoricUkPropertyPeriodSummariesRawData(nino)
+      val validator = validatorFactory.validator(nino)
 
-      val requestHandler = RequestHandlerOld
-        .withParser(parser)
+      val requestHandler = RequestHandler
+        .withValidator(validator)
         .withService({ req =>
           service.listPeriodSummaries(req, propertyType)
         })
-        .withResultCreator(ResultCreatorOld.hateoasListWrapping(hateoasFactory)((_, _) =>
+        .withResultCreator(ResultCreator.hateoasListWrapping(hateoasFactory)((_, _) =>
```

Comment:
Can directly import hateoaslistWrappign so that this line becomes ` .withResultCreator(hateoasListWrapping(hateoasFactory)((_, _) =>`